docs/source/tensorbay_cli/cli_commands.rst

```diff
@@ -308,6 +308,12 @@ Show commit logs in oneline format.
 
    gas log --oneline tb:[dataset_name]
 
+Show a text-based graphical representaion of commit history
```

Comment:
the `--graph` feature may be broken if gas log for multi-branched is implemented.
I think it is better to implement multi-branches log first.
then implement `--graph` logic for both single and multi branches cases